test/e2e/instrumentation-hook/instrumentation-hook.test.ts

```diff
@@ -0,0 +1,61 @@
+import { createNextDescribe } from 'e2e-utils'
+import { check } from 'next-test-utils'
+
+createNextDescribe(
+  'instrumentation-hook',
+  {
+    files: __dirname,
+    nextConfig: {
+      experimental: {
+        instrumentationHook: true,
+      },
+    },
+    forcedPort: 3000,
```

Comment:
Why are we forcing port 3000 here? This can cause issues in CI if the port isn't available